src/views/LocusGeneDiseaseView.vue

```diff
@@ -459,10 +448,7 @@ export default {
           <tr
             v-if="
               locusGeneDiseaseData.molecular_mechanism &&
```

Comment:
once the suggestion on line 451 "locusGeneDiseaseData.molecular_mechanism?.evidence" is added, this condition wont be required, please delete this condition